packages/database/src/data-source.ts

```diff
@@ -27,3 +28,19 @@ export const container = async (fn: () => Promise<void>) => {
   await fn()
   await db.close()
 }
+
+export const container2 = async (fn: () => Promise<any>): Promise<any> => {
```

Comment:
as discussed, confirm that we can remove the original `container` function, and rename `container2` to `container`

Reply:
yep. resolved with [74a3e1e](https://github.com/modtree/modtree/pull/27/commits/74a3e1ed68050cc5f191dc9e54d9aa6546696550)